### ISSUE TITLE
Nunjucks rendering issue

### ISSUE THREAD

**Issue by mvolkmann** [11ty/eleventy#1145]
What am I doing wrong in this snippet from one of my Nunjucks macros?
`entry.children` is either an array or it is undefined.
This outputs a link with `entry.title` as its text.
But it also outputs the literal string "\&#x25b6;" AND the literal string "\</a>".

```njk
<a class="nav-link" href="{{ entry.url | url }}" target="frame">
  {{ entry.title }}
  {% if entry.children %}
    &#x25b6;
  {% endif %}
</a>
```

**Comment by pdehaan** [comment]
Not your exact setup, but I don't think I'm reproducing in v0.10 with the following standalone README.njk

```liquid
---
entry:
  url: https://11ty.dev/docs
  title: "Behold the docs!"
  children: []
---

<a class="nav-link" href="{{ entry.url | url }}" target="frame">
  {{ entry.title }}
  {% if entry.children %}
    &#x25b6;
  {% endif %}
</a>
```

OUTPUT:

```html
<a class="nav-link" href="https://11ty.dev/docs" target="frame">
  Behold the docs!
  
    &#x25b6;
  
</a>
```

(Looks like even an empty array is truthy, so I'm still seeing the `&#x25b6;` in my example output.)

Or, if you're just objecting to just the literal `</a>` appearing in your output, I'm wondering if you're maybe missing a [`safe` filter](https://mozilla.github.io/nunjucks/templating.html#safe) somewhere in your Nunjucks tree which is causing the HTML to be escaped.

**Comment by mvolkmann** [comment]
I tried hard to figure this out, but failed. I gave up and switched to using a JavaScript-based layout inspired by https://gitlab.com/reubenlillie/eleventy-dot-js-blog/. If anyone wants to see my solution, it is at https://github.com/mvolkmann/blog. Thanks @reubenlillie!

See the file `topics.11ty.js` which is used as the layout for `topics.md`.

The running site is at https://mvolkmann.github.io/blog/.
